scripts/scriptutils/part_retrieval.py

```diff
@@ -349,26 +354,31 @@ def package_parts_inventory(package: str, targets: List[str] = None) -> PackageI
     inventory = PackageInventory()
 
     # import FASTAs and GenBank
-    for file in sorted(itertools.chain(*(glob.glob(os.path.join(package, f'*{ext}')) for ext in GENETIC_DESIGN_FILE_TYPES['FASTA']))):
+    for file in sorted(itertools.chain(
+            *(glob.glob(os.path.join(package, f'*{ext}')) for ext in GENETIC_DESIGN_FILE_TYPES['FASTA']))):
         is_igem_cache = os.path.basename(file) == IGEM_FASTA_CACHE_FILE
         prefix = iGEM_SOURCE_PREFIX if is_igem_cache else package_stem(package)
         with open(file) as f:
             import_file = ImportFile(file, file_type='FASTA', namespace=prefix)
             for record in SeqIO.parse(f, "fasta"):
                 identity = id_map[record.id] if record.id in id_map else accession_to_sbol_uri(record.id, prefix)
+                print(f'Fasta Part identity is {identity}')
```

Comment:
Does this need to be added to the standard output? It's pretty chatty already.

Reply:
I was meaning to comment it out. It helped me when trying to find out what set/list/dict has what elements but it is not needed for standard output.